firefish/fluids.py

```diff
@@ -0,0 +1,54 @@
+"""
+This module provides shortcuts to commonly used fluids
+"""
+
+import enum
+
+from firefish.case import (
+    FileName)
+
+class Fluid(enum.Enum):
+    """
+    An enumeration of commonly used fluids
+
+    AIR generates the recommended OpenFoam thermophysicalProperties for air.
+
+    DIMENSIONLESS_AIR generates a normalised gas whith gamma=7/5 and with
+    the property that at 1 temperature unit the speed of sound is 1
+    velocity unit"""
```

Comment:
Tiny nitpick here: the convention for multiline docstrings is to have a blank line at the end:

```python
class Fluid(enum.Enum):
    """This is
    a multiline docstring

    """
    AIR=0

etc, etc.